### PR TITLE
[IMP] web: editable list view column is not sort when cell is in edit…

### PR DESCRIPTION
**Pull request by dep-odoo**
Issue : https://www.odoo.com/web#id=749480&view_type=form&model=project.issue&action=963&active_id=681&menu_id=4720

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by aab-odoo** [comment]
Hello,

I don't know if this is the correct fix, but you should write a test to reproduce the issue anyway.

Regards

**Comment by dep-odoo** [comment]
Hello @aab-odoo 

test case was added please check

Thanks

File: addons/web/static/tests/views/list_tests.js [hunk 1]
```diff
@@ -861,6 +881,7 @@ QUnit.module('Views', {
         assert.ok(form.$('tbody tr:eq(2) td input').val(),
             "Value 2 should be third (shouldn't be sorted)");
 
+        form.$('.o_form_sheet_bg').click();
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_tests.js]: ?

Comment (dep-odoo) [reply addons/web/static/tests/views/list_tests.js]: When list view is editable (active editing), We've now prevented sorting. So to sort, Now I click on this element to focus out from active editing.

Comment (aab-odoo) [reply addons/web/static/tests/views/list_tests.js]: makes sense :) Thanks

**Comment by aab-odoo** [comment]
Merged: https://github.com/odoo/odoo/commit/9f8efb9d164a6995299c9e76926effc2bb3aae6d
Thanks!

**Pull request closed without merging**